pytest.ini

```diff
@@ -1,2 +1,2 @@
 [pytest]
-norecursedirs = pytest_rts/tests/helper_project
+norecursedirs = pytest_rts/tests/helper_project pytest_rts/utils
```

Comment:
Why do you need here `pytest_rts/utils`? Pytest is usually quite good with test autodiscovery. At least add a comment above.